File: Modul 4/Soal 1/PRAK401-2410817220018-ANGGRAENI DWI ZAHRA.py
kelipatan = int(input("Masukkan kelipatan: "))
simbol = input("Masukkan simbol: ")

for i in range(1, 51):
    if i % kelipatan == 0:
        print(simbol, end=" ")
    else: 
         print(i, end=" ")

File: Modul 4/Soal 1/PRAK401-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
int main () {
    int kelipatan;
    char simbol;

    printf("Masukkan kelipatan: ");
    scanf("%d", &kelipatan);
    printf("Masukkan simbol: ");
    scanf(" %c", &simbol);

    int i;

    for (i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {
            printf("%c ", simbol);
        } else {
            printf ("%d ", i);
        }
    }

    return 0;
}